### ISSUE TITLE
@elsecan / @elsecannot functionality

### ISSUE THREAD

**Issue by ngaugler** [laravel/framework#13048]
Is there a way to effectuate @else and @can into blades?  I have run into a situation where I need to leverage @elsecan() but do not see this functionality in Laravel.  Below is a basic example:

```php
@can('read-write')
<li>Read/Write</li>
@elsecan('write')
<li>Write</li>
@elsecan('read')
<li>Read</li>
@else
<li>None</li>
@end
```

If a new feature is added for @elsecan, it would make sense to add @elsecannot at the same.  

---Edit---
I ended up creating my own directive in AppServiceProvider::boot(), but this is probably a useful directive for others.

```php
Blade::directive('elsecan', function($expression) {
    return "<?php elseif (Gate::check{$expression}): ?>";
});
```

**Comment by themsaid** [comment]
@GrahamCampbell this too can be closed

**Comment by GrahamCampbell** [comment]
Thank you.

**Comment by GrahamCampbell** [comment]
I future, please format PR bodies so they actually contain the text `Closes #13048` ;)

**Comment by themsaid** [comment]
I started doing that already in my latest PRs, making your job a bit easier ;)
